src/Crypto/JWT.hs

```diff
@@ -279,17 +283,22 @@ class HasAllowedSkew s where
   allowedSkew :: Lens' s NominalDiffTime
 class HasAudiencePredicate s where
   audiencePredicate :: Lens' s (StringOrURI -> Bool)
+class HasIssuerPredicate s where
+  issuerPredicate :: Lens' s (StringOrURI -> Bool)
 
 instance HasJWTValidationSettings a => HasAllowedSkew a where
   allowedSkew = jwtValidationSettingsAllowedSkew
 instance HasJWTValidationSettings a => HasAudiencePredicate a where
   audiencePredicate = jwtValidationSettingsAudiencePredicate
+instance HasJWTValidationSettings a => HasIssuerPredicate a where
+  issuerPredicate = jwtValidationSettingsIssuerPredicate
 
 defaultJWTValidationSettings :: JWTValidationSettings
 defaultJWTValidationSettings = JWTValidationSettings
   defaultValidationSettings
   0
   (const False)
+  (const False)
```

Comment:
Default should be `const True`.  Default for audience predicate is `const False` because of requirement that client MUST identify with a value in `"aud"` claim if that claim is present (https://tools.ietf.org/html/rfc7519#section-4.1.3).  There is no such requirement for the `"iss"` claim, so default should be to allow any value.